src/konnektor/network_planners/generators/maximal_network_generator.py

```diff
@@ -22,14 +22,18 @@ def __init__(
         n_processes: int = 1,
     ):
         """
-        The `MaximalNetworkGenerator` builds for given set of `Component` s a fully connected graph under the assumption each `Component` can be connected to another.
-        The `Transformation` s of this graph are realized as `AtomMapping` s of pairwise `Component` s. If not all mappings can be created, it will ignore the mapping failure, and return a nearly fully connected graph.
-
-        Note: This approach is not very suitable for Free Energy calculations in application cases. However, this approach is very important, as all above approaches use this as an initial solution, they filter down to gain the desired design.
-
-
-        This class is recommended as initial_edge_lister for other approaches.
-        > **Note**: the `MaximalNetworkGenerator` is parallelized and the number of CPUs can be given with  `n_processes`.
+        The `MaximalNetworkGenerator` builds a fully connected graph for given set of `Component` s.
+        It assumes each `Component` can be connected to every other `Component`.
+        The `Transformation` s of this graph are realized as `AtomMapping` s of pairwise `Component` s.
+        If not all mappings can be created, it will ignore the mapping failure and return a nearly fully connected graph.
+
+        Note: This approach is not recommended for Free Energy calculations in application cases, as it is
+	very computationally expensive.
+        However, this approach is very important, as all other approaches use the Maximal Network as an initial solution,
+        then remove edges to achieve the desired design.
+
+        This class is recommended as an `initial_edge_lister` for other approaches.
+        > **Note**: the `MaximalNetworkGenerator` is parallelized and the number of CPUs can be given with `n_processes`.
```

Comment:
Why is this note differently formatted than the other note?

Reply:
I think back-to-back "note" callout boxes is confusing. I'm removing the second callout box, but willing to change this description if someone has an opinion.